### PR TITLE
Enable CI for smoke test and pycloudstack runs

### PR DESCRIPTION
**Pull request by bprashan**

### CODE REVIEW

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -38,26 +38,28 @@ pipeline{
...
-                        }
-                        sh "chmod +x $tdx_linux_stack_script"
-                        def statusCode = sh(script: "./$tdx_linux_stack_script --setuptdx", returnStatus:true)
-                        if (statusCode != 0 ){
-                           if(statusCode == 3){
-                                echo "Build machine will be restarted..."
-                                isRestartrequired = true
-                            }else {
-                                sh "exit 1"
+                                tdx_linux_stack_script = 'tdx_centos_linux_stack.sh'
```
Comment (jinengandhi-intel) [inline ci/Jenkinsfile]: Fix the indentation. It seems to have 2 tabs.

File: ci/Jenkinsfile [hunk 2]
```diff
@@ -38,26 +38,28 @@ pipeline{
...
+                            if (statusCode != 0 ){
+                               if(statusCode == 3){
+                                    echo "Build machine will be restarted..."
+                                    isRestartrequired = true
+                                }else {
+                                    sh "exit 1"
+                               }
                             }
-                       }
+                        }
```
Comment (jinengandhi-intel) [inline ci/Jenkinsfile]: Fix the indentation. Should be on par with the above If statement.

**Review by jinengandhi-intel — CHANGES_REQUESTED** [review]

File: ci/Jenkinsfile (hunk 1 above)
Comment (bprashan) [reply ci/Jenkinsfile]: updated

File: ci/Jenkinsfile (hunk 2 above)
Comment (bprashan) [reply ci/Jenkinsfile]: updated

**Review by jinengandhi-intel — APPROVED** [review]

**Pull request merged**